spec/components/meta_tags_spec.rb

```diff
@@ -31,6 +31,30 @@ def example_document_for(schema_name, example_name)
     assert_meta_tag('govuk:schema-name', 'publication')
   end
 
+  it "renders the canonical URL in a meta tag if requested" do
+    render_component(
+      content_item: { base_path: "/test" },
+      canonical_path: true,
+    )
+
+    assert_meta_tag('canonical', 'http://www.dev.gov.uk/test')
+  end
+
+  it "renders the canonical URL in a meta tag if overridden" do
+    render_component(
+      content_item: { base_path: "/test" },
+      canonical_path: "/test2",
+    )
+
+    assert_meta_tag('canonical', 'http://www.dev.gov.uk/test2')
+  end
+
+  it "doesn't renders the canonical URL in a meta tag if not requested" do
+    assert_empty render_component(content_item: {
+      base_path: "/test"
+    }).strip
+  end
```

Comment:
I'm sorry I was all set to give the ✅ but there's some quote inconsistencies

Reply:
Yeah... I've made them all single quotes I think... I like to think they were consistent with the inconsistent style of the file...